src/script/lua_api/l_vmanip.cpp

```diff
@@ -91,7 +91,7 @@ int LuaVoxelManip::l_set_data(lua_State *L)
 	MMVManip *vm = o->vm;
 
 	if (!lua_istable(L, 2))
-		return 0;
+		throw LuaError("set_data called with missing parameter");
```

Comment:
"VoxelManip:set_data() called with ..."

Same for the other throw LuaError lines?

Reply:
Thanks, this makes it clear that the message refers to vmanip. I did not add the parenthesis because I think they would be confusing when the message is about missing parameters.

Reply:
Oops yes this clarification is important.